packages/astro/test/units/dev/dev.test.js

```diff
@@ -56,7 +56,7 @@ describe('dev container', () => {
 					<html>
 						<head><title>{name}</title></head>
 						<body class="one">
-							<Header title={name} />
+							<Header />
```

Comment:
```suggestion
							<Header title={name} />
```

The `title` prop is still used in tests.